### PR TITLE
Propose fix some typos

### PR DESCRIPTION
**Pull request by jeis2497052**
Does this help fixing a couple of typo's ?

### CODE REVIEW

File: _i18n/en/_posts/2018-01-13-bus.md [hunk 1]
```diff
@@ -3,15 +3,15 @@ author: tungfa
...
-original-author: t0dd  
+original-author: t0dd
 original-link: https://www.dashforcenews.com/happens-cryptocurrency-get-hit-bus/
 ---
 
 Or, how to manage volatility's ultimate endstate: Death.
 
 In a previous article, I shared some guidance on how to introduce new people to Dash and other cryptocurrencies: "[Introducing FOMO Friends and Family to Cryptocurrencies](https://www.dashforcenews.com/introducing-fomo-friends-family-cryptocurrencies/)". Within that article, we touched upon end-of-life matters. In this article, we expand that topic and give it a bit more consideration.
 
-Answer this question: What happens to your cryptocurrency hodlings if you get hit by a bus?
```
Comment (nmarley) [inline _i18n/en/_posts/2018-01-13-bus.md]: I think this particular "typo" might be intentional. ;-)

Comment (jeis2497052) [reply _i18n/en/_posts/2018-01-13-bus.md]: OK, so revert this one ?

Comment (nmarley) [reply _i18n/en/_posts/2018-01-13-bus.md]: Well, let's wait to see what @tungfa says on it.

Comment (tungfa) [reply _i18n/en/_posts/2018-01-13-bus.md]: hey guys
"holdings" sounds good to me ! (am not native english)

**Comment by nmarley** [comment]
@jeis2497052 Looks good, thank you!

@mrleewong Feel free to merge this typos fix whenever you can, I will let you handle it since you are primary maintainer of the site.

**Pull request merged**